src/python/zigopt/experiment/util.py

```diff
@@ -1,14 +1,14 @@
 # Copyright © 2022 Intel Corporation
 #
 # SPDX-License-Identifier: Apache License 2.0
+from libsigopt.sigoptaux.constant import DEFAULT_USE_SPE_BEYOND_THIS_MANY_DIMENSIONS
```

Comment:
Eventually, constants like this would probably need to exist inside of `libsigopt.validation`

Reply:
We can consolidate when validation module is completed.